stock_move_value/models/stock_move.py

```diff
@@ -55,18 +55,18 @@ def _action_done(self, cancel_backorder=False):
                 lambda r: r.quantity > 0
             )
             origin_values[move.id] = {
-                "remaining_qty": origin_svls.remaining_qty,
-                "remaining_value": origin_svls.remaining_value,
+                "quantity": origin_svls.quantity,
+                "unit_cost": origin_svls.unit_cost,
```

Comment:
As per the original design, it should be `value` instead of `unit_cost` in my opinion.

Reply:
Right.   I will fix.